src/Algebra/Bundles.agda

```diff
@@ -341,6 +341,30 @@ record DistributiveLattice c ℓ : Set (suc (c ⊔ ℓ)) where
 -- Bundles with 2 binary operations & 1 element
 ------------------------------------------------------------------------
 
+record RawNearSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
+  infixl 7 _*_
+  infixl 6 _+_
+  infix  4 _≈_
+  field
+    Carrier    : Set c
+    _≈_        : Rel Carrier ℓ
+    _+_        : Op₂ Carrier
+    _*_        : Op₂ Carrier
+    0#         : Carrier
+
+  +-rawMonoid : RawMonoid c ℓ
```

Comment:
I would probably open this publicly and export `+-rawMagma` as well?

Reply:
Good catch. Fixed. Thanks.